.github/workflows/build.yml

```
@@ -117,13 +142,22 @@
+    name: "macOS Release (${{ matrix.noscrypt_name }})"
+    strategy:
+      matrix:
+        noscrypt: ["OFF", "ON"]
+        include:
+          - noscrypt: "OFF"
+            noscrypt_name: "secp256k1"
+          - noscrypt: "ON"
+            noscrypt_name: "noscrypt"
     
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)